### PR TITLE
New package: freedoom-0.13.0

### PR DESCRIPTION
**Pull request by lilmike**
#### Testing the changes
- I tested the changes in this PR: **YES**
                                                                                                                                
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/freedoom/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="https://github.com/$pkgname/$pkgname/releases/download/v$version/$pkgname-$version.tar.xz"
+checksum=bad733695664c699fc7f02c833916c889b080fadc3b503c40f0aaba3bf637380
+
+pre_build() {
+	# CPP doesn't seem to be used for a c preprocessor in this makefile (?), so replacing the value in the makefile breaks the build.
+	make_build_args="CPP=${XBPS_BUILDDIR}/freedoom-${version}/scripts/simplecpp"
+}
+
+post_install() {
+	vlicense ${XBPS_BUILDDIR}/freedoom-${version}/COPYING.adoc
```
Comment (classabbyamp) [inline srcpkgs/freedoom/template:20]:
```suggestion
	vlicense COPYING.adoc
```
you're already in that dir (wrksrc)

File: srcpkgs/freedoom/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+maintainer="Michael Taboada <voidpkgs@forwardme.email>"
+license="BSD-3-Clause"
+homepage="https://freedoom.github.io/"
+distfiles="https://github.com/$pkgname/$pkgname/releases/download/v$version/$pkgname-$version.tar.xz"
+checksum=bad733695664c699fc7f02c833916c889b080fadc3b503c40f0aaba3bf637380
+
+pre_build() {
+	# CPP doesn't seem to be used for a c preprocessor in this makefile (?), so replacing the value in the makefile breaks the build.
+	make_build_args="CPP=${XBPS_BUILDDIR}/freedoom-${version}/scripts/simplecpp"
+}
```
Comment (classabbyamp) [inline srcpkgs/freedoom/template:17]:
```suggestion
make_build_args="CPP='scripts/simplecpp'"
```
and put it after build_style

File: srcpkgs/freedoom/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+pkgname=freedoom
+version=0.12.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="asciidoc deutex python3 python3-Pillow"
+short_desc="Free game based on the Doom engine"
+maintainer="Michael Taboada <voidpkgs@forwardme.email>"
+license="BSD-3-Clause"
+homepage="https://freedoom.github.io/"
+distfiles="https://github.com/$pkgname/$pkgname/releases/download/v$version/$pkgname-$version.tar.xz"
```
Comment (classabbyamp) [inline srcpkgs/freedoom/template:11]: don't use $pkgname in distfiles

File: srcpkgs/freedoom/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'freedoom'
+pkgname=freedoom
+version=0.12.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="asciidoc deutex python3 python3-Pillow"
```
Comment (classabbyamp) [inline srcpkgs/freedoom/template:6]:
```suggestion
hostmakedepends="asciidoc deutex python3-Pillow"
```
python3 is redundant if python3-Pillow is there

File: srcpkgs/freedoom/template (hunk 2 above)
Comment (lilmike) [reply srcpkgs/freedoom/template]: Using scripts/simplecpp breaks when make chdirs to another directory, no idea how the makefile normally works at all lol.

File: srcpkgs/freedoom/template [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'freedoom'
+pkgname=freedoom
+version=0.12.1
+revision=1
+build_style=gnu-makefile
+make_build_args="CPP=${XBPS_BUILDDIR}/freedoom-${version}/scripts/simplecpp" # Setting to scripts/simplecpp will fail when make changes to another directory
```
Comment (classabbyamp) [inline srcpkgs/freedoom/template:6]:
```suggestion
make_use_env=yes
```

**Comment by lilmike** [comment]
This is also still relevant.

-Michael.

**Comment by MIvanchev** [comment]
Hey, great work here, you need to update the MR title when updating the template file. The versions don't match.

**Comment by lilmike** [comment]
Thanks, I've done this.

**Comment by lilmike** [comment]
This is still relevant.




-- 
Sorry about the test, it's just a me thing.
Michael Taboada, Creator of Games. Does that make me a god, well, sorta.
My pgp key: 79BC390E8864CD71305D4DD606CD2197A6EE5ED7
My website: https://michaels.world
2MB website: https://2mb.games
Don't believe everything you read on the internet. I might not be real, you might not be real, and this email certainly isn't real.
Sent from my supergenius...
stationary bicycle

**Pull request closed without merging**